src/ai/assistants/assistants.ts

```diff
@@ -0,0 +1,79 @@
+import OpenAI from 'openai';
+
+import SdkBase from '../../sdkBase';
+import Ainft721Object from '../../ainft721Object';
+import { AssistantCreateParams, AssistantTransactionResult } from '../../types';
+import {
+  buildSetTransactionBody,
+  validateAi,
+  validateObject,
+  validateObjectOwnership,
+  validateService,
+  validateServiceName,
+  validateToken,
+} from '../../util';
+
+export default class Assistants extends SdkBase {
```

Comment:
(suggestion) Instead of inheriting from SdkBase, how about adding an explicit constructor e.g.
```
  constructor(ain: Ain, ainize: Ainize) {
    this.ain = ain;
    this.ainize = ainize;
};
```
for readability?

Reply:
Done! Thanks!